arlo-client/src/setupProxy.js

```diff
@@ -16,6 +16,7 @@ module.exports = function(app) {
   app.use(proxy('/election/new', { target }))
   app.use(proxy('/election/*/audit/**', { target }))
   app.use(proxy('/election/*/jurisdiction/**', { target }))
+  app.use(proxy('/election/*/jurisdictions/**', { target }))
```

Comment:
Should we use `…/jurisdiction/…` for this? cc @benadida @jonahkagan

Reply:
Do we have both `jurisdiction` and `jurisdictions`? If so would be good to standardize, yes.

Reply:
@eventualbuddha @benadida Interestingly, now I take a closer look, the actual endpoint we were using for the single page flow is `.../jurisdictions` and the new one just has `/file` appended to it. It doesn't look like we *have* a singular jurisdiction endpoint proper. :thinking: I wonder why it worked with this file as it was. Maybe I'm missing something?

Reply:
In a separate PR, we should change `jurisdictions` to `jurisdiction` and then leave this line out.

Reply:
https://github.com/votingworks/arlo/pull/368